tests/tt_eager/python_api_testing/unit_testing/test_bert_ops.py

```diff
@@ -20,612 +20,612 @@
-            compute_kernel_config=compute_kernel_config,
-        )
-
-    if out_sharded:
-        output_t = ttl.tensor.sharded_to_interleaved(output_t, interleaved_mem_config_L1)
-
-    pt_out = in0 @ in1
-
-    if has_bias:
-        pt_out = pt_out + bias
-
-    if activation != None:
-        pt_out = torch.nn.functional.gelu(pt_out)
-    tt_out = tt2torch_tensor(output_t)
-
-    passing, output = comp_pcc(pt_out, tt_out)
-    logger.info(output)
-    assert passing
-
-
-def not_fit_l1(M, K, N, fp32):
-    return (M * K + K * N > 5000000) and (fp32 == True)
-
-
-@pytest.mark.skipif(is_grayskull(), reason="GS does not support fp32")
-@pytest.mark.parametrize("packer_l1_acc", [True, False], ids=["pack_l1", "no_pack_l1"])
-@pytest.mark.parametrize("fp32_acc_mode", [True, False], ids=["fp32", "no_fp32"])
-@pytest.mark.parametrize(
-    "fidelity",
-    [
-        ttl.tensor.MathFidelity.LoFi,
-    ],
-    ids=["LoFi"],
-)
-@pytest.mark.parametrize("has_bias", [True, False], ids=["bias", "no_bias"])
-@pytest.mark.parametrize(
-    "in1_in_dram, out_sharded, in0_sharded, M, K, N, activation",
-    [
-        # in1-L1-fusedQKV
-        (False, True, True, 1536, 1024, 3072, None),  # both sharded
-        # in1-dram-fusedQKV
-        (True, True, True, 1536, 1024, 3072, None),
-        # in1-L1-selfout
-        (False, True, True, 1536, 1024, 1024, None),
-        # in1-dram-selfout
-        (True, True, True, 1536, 1024, 1024, None),
-        # in1-L1-ff1
-        (False, True, True, 1536, 1024, 4096, (ttl.tensor.FusibleActivation.GELU, True)),
-        # in1-dram-ff1
-        (True, True, True, 1536, 1024, 4096, (ttl.tensor.FusibleActivation.GELU, True)),
-        # in1-L1-ff1 - no Gelu
-        (False, True, True, 1536, 1024, 4096, None),
-        # in1-dram-ff1 - no Gelu
-        (True, True, True, 1536, 1024, 4096, None),
-        # in1-L1-ff2
-        (False, True, True, 1536, 4096, 1024, None),
-        # in1-dram-ff2
-        (True, True, True, 1536, 4096, 1024, None),
-    ],
-)
-def test_bert_linear_batch4(
-    device,
-    in0_sharded,
-    out_sharded,
-    in1_in_dram,
-    M,
-    K,
-    N,
-    fidelity,
-    has_bias,
-    activation,
-    packer_l1_acc,
-    fp32_acc_mode,
-    function_level_defaults,
-):
-    for i in range(1):
-        logger.info(i)
-        if not not_fit_l1(M, K, N, fp32_acc_mode):
-            run_bert_linear_batch4(
-                device,
-                in0_sharded,
-                out_sharded,
-                in1_in_dram,
-                M,
-                K,
-                N,
-                fidelity,
-                has_bias,
-                activation,
-                packer_l1_acc,
-                fp32_acc_mode,
-                function_level_defaults,
-            )
-        else:
-            logger.warning("L1 cannot fit large tensors in fp32 mode")
+# @pytest.mark.skipif(is_wormhole_b0(), reason="Unsupported parallelizations for WH B0")
```

Comment:
I wonder if these commented-out test cases relate to this single-core Matmul benchmark.

Reply:
will revert back